### DISCUSSION TITLE
GDPS Question.

### DISCUSSION THREAD

**Discussion by ghost** [zmxhawrhbg/gdrpc#5] [Q&A]
Not a bug, but I need to know something. If you want to use this on a GDPS, do you need to recompile the source code after changing stuff?

**Comment by qimiko** [comment]
as of right now, you'll have to change the source code and recompile
this should change in the next update (whenever that is) but i'm still in the bug testing stage of it

**Comment by ghost** [comment]
oh ok my GDPS is planned to start about like next week or so I will just do to, i only have a compiler for python as im a python developer (along with web coding) so what should i use for this

**Comment by qimiko** [comment]
you'll need to install visual studio (i recommend 2019) and cmake (visual studio 2019 has built in cmake support)
from there just follow the steps outlined in https://github.com/zmxhawrhbg/gdrpc#compiling to compile this

**Comment by ghost** [comment]
ah ok i already have Visual Studio and Visual Studio Code installed, ill just check on the vs installer if i see anything! ill just reopen this ticket if im having problems if i dont assume it worked thanks
